src/axom/inlet/Proxy.cpp

```diff
@@ -49,5 +49,48 @@ Proxy Proxy::operator[](const std::string& name) const
   return (*m_table)[name];
 }
 
+const axom::sidre::Group* Proxy::sidreGroup() const
+{
+  if(m_table != nullptr)
+  {
+    return m_table->sidreGroup();
+  }
+  else if(m_field != nullptr)
+  {
+    return m_field->sidreGroup();
+  }
+  else if(m_func != nullptr)
+  {
+    SLIC_ERROR("[Inlet] Cannot retrieve the sidre::Group for a Function");
```

Comment:
Can this print the path for the `Proxy`?  
(This might only makes sense if `Proxy` stores a path independently of the `sidre::Group`).

EDIT: Looking that the member fields for `Proxy` -- looks like the answer is "No. That information is not available".

Reply:
This could be added since everywhere that constructs a Proxy has path information - this should probably be optional though, since the implicit Table -> Proxy conversion appears to be useful: https://github.com/LLNL/axom/issues/453#issue-802425925